common/src/main/java/tachyon/TachyonURI.java

```diff
@@ -332,10 +337,9 @@ public boolean hasScheme() {
    * @param slashed if the path starts with a slash.
    * @return true if it is a windows path, false otherwise
    */
-  private boolean hasWindowsDrive(String path, boolean slashed) {
+  public static boolean hasWindowsDrive(String path, boolean slashed) {
     int start = slashed ? 1 : 0;
-    return WINDOWS
-        && path.length() >= start + 2
+    return path.length() >= start + 2
```

Comment:
This is important, could you please add this comment into the javadoc of this method?